osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/factories/DlmsConnectionHelper.java

```diff
@@ -149,6 +149,7 @@ private void createAndHandleConnectionForDevice(
             messageMetadata,
             permit,
             taskForConnectionManager);
+        return;
```

Comment:
Without this return always a exeception is thrown when a invocation counter is out-of-sync